src/bind.ts

```diff
@@ -72,14 +72,14 @@ function bindElements(root: Element | ShadowRoot) {
 function handleEvent(event: Event) {
   const el = event.currentTarget as Element
   for (const binding of bindings(el)) {
-    if (event.type === binding.type && controllers.has(binding.tag)) {
-      type EventDispatcher = Element & Record<string, (ev: Event) => unknown>
-      const controller = el.closest(binding.tag) as EventDispatcher
-      if (controller && typeof controller[binding.method] === 'function') {
+    if (event.type === binding.type) {
+      type EventDispatcher = HTMLElement & Record<string, (ev: Event) => unknown>
+      const controller = el.closest<EventDispatcher>(binding.tag)!
```

Comment:
Why this change?

Reply:
This specific line is mostly an artefact of refactoring.